From 28f48bcdc82eba2a8afa079c1ee014a441fdc9da Mon Sep 17 00:00:00 2001
From: Kacper Lewandowski <kacper.lewandowski540@gmail.com>
Date: Thu, 1 Jun 2023 18:02:29 +0200
Subject: [PATCH] cleanup + working Makefile

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 12d858f..2cb65bb 100644
--- a/Makefile
+++ b/Makefile
@@ -14,7 +14,7 @@ ALEX_OPTS  = --ghc
 
 # Default goal.
 
-all : Atuan/Test
+all : Atuan/Interpreter
 
 # Rules for building the parser.
 
@@ -27,7 +27,7 @@ Atuan/Abs.hs Atuan/Lex.x Atuan/Par.y Atuan/Print.hs Atuan/Test.hs : Atuan.cf
 %.hs : %.x
 	${ALEX} ${ALEX_OPTS} $<
 
-Atuan/Test : Atuan/Abs.hs Atuan/Lex.hs Atuan/Par.hs Atuan/Print.hs Atuan/Test.hs
+Atuan/Interpreter : Atuan/Abs.hs Atuan/Lex.hs Atuan/Par.hs Atuan/Print.hs Atuan/Test.hs Atuan/CollectTypes.hs Atuan/AlgorithmW.hs Atuan/Translate.hs
 	${GHC} ${GHC_OPTS} $@
 
 # Rules for cleaning generated files.
-- 
2.34.1

